.golangci.yml

```diff
@@ -49,9 +49,9 @@ linters:
       - linters:
           - staticcheck
         text: QF1001
-      - linters:
-          - staticcheck
-        text: QF1003
+      # - linters:
```

Comment:
Any reason to comment this out instead of removing?

Reply:
Should have been removed. Was commented during testing.

Reply:
Will update this today whilst resolving the conflict post-rebase.

Reply:
Resolved, but I did need to drop QF1001 back in since two additional items for that have popped up.